client/src/main/scala/org/http4s/client/middleware/Retry.scala

```diff
@@ -56,9 +60,15 @@ object Retry {
       }
 
     def nextAttempt(req: Request[F], attempts: Int, duration: FiniteDuration)
-                   (implicit F: Async[F], executionContext: ExecutionContext): F[DisposableResponse[F]] =
-      // TODO honor Retry-After header
-      scheduler.sleep_[F](duration).run >> prepareLoop(req.withEmptyBody, attempts + 1)
+                   (implicit F: Async[F], executionContext: ExecutionContext): F[DisposableResponse[F]] = {
+      val sleepDuration = req.headers.get(`Retry-After`).map { h =>
```

Comment:
`Retry-After` is a response header.  We'll need to get that up in the case on line 39.

Reply:
OOPS 👅